EveryVoiceTTS/EveryVoice
# Dev.ej/460

### PR Goal? 

Fix the bug where a literal `/` in text would cause a crash, by replacing it with `_SLASH_` when the characters are joined with  `/`.

### Fixes? 

Fixes #460
Fixes #540

### Feedback sought? 

This `_SLASH_` is not just in internal state, it's also saved in `preprocessed/filelist.psv`, which means it's ultimately user visible. Because of that, I would like a strong agreement on what string to use here. Is `_SLASH_` OK or should I use `<SLASH>` or something else? I'm not worried about collisions, but whatever choice we make now will be permanent because changing it in the future will be a breaking change.

Beyond that, regular sanity checking.

### Priority? 

beta

### Tests added? 

yes

### How to test? 

Follow https://github.com/EveryVoiceTTS/EveryVoice/issues/460#issuecomment-2455849389
and see that `everyvoice preprocess` doesn't crash.

Or use `everyvoice/tests/data/metadata_slash_pipe.psv` in the wizard and see that `everyvoice preprocess` doesn't crash.

### Confidence? 

High for the coding, low for the choice of substitution string.

### Version change? 

no

### Related PRs? 

none
